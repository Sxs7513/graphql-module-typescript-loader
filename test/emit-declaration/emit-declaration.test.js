const fs = require('fs')
const compiler = require('../compiler.js')

test('Can emit valid declaration', async () => {
    try {
        await compiler({ entry: require.resolve('./index.js') })
    } catch (e) {
        console.log(e)
    }

    const declaration = fs.readFileSync(
        require.resolve('./test.graphql.d.ts'),
        'utf-8'
    )

    expect(declaration).toMatchSnapshot()
})