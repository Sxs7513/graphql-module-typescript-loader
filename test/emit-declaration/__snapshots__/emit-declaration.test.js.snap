// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Can emit valid declaration 1`] = `
"// This file is automatically generated.
// Please do not change this file!
export const test_mutation_1: any;
export const test_mutation_2: any;
export const test_mutation_3: any;
export const test_query_1: any;
export const test_query_2: any;
export const test_query_3: any;"
`;
